plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -426,13 +462,18 @@ export class MongoDBInstrumentation extends InstrumentationBase<
     // mongodb is using "tick" when calling a callback, this way the context
     // in final callback (resultHandler) is lost
     const activeContext = context.active();
+    const config = this._getConfig();
+    const executeFunction = this._handleExecutionResult;
```

Comment:
A more standard way of doing that is keeping `this` as a const and calling `_handleExecutionResult` with it.

Reply:
Done with you ;)